### PR TITLE
refactor: product card and style sheets

### PR DESCRIPTION
**Pull request by TejasShekar**
Preview Link here : https://p8-games-git-authentication-tejasshekar.vercel.app/products

Refer individual commits for changelogs

### CODE REVIEW

**Review by kotesh-arya — COMMENTED** [review]
Responsitivity works perfect and CSS improvements, Looks Good for Merging 👍🏻 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by kotesh-arya — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+// import { useState } from "react";
 import { useCart } from "../context/cart-context";
 import { calcQuantity } from "../utils/cartSummary";
 import { useAuth } from "../context";
 import { toast } from "react-toastify";
 
 export const Header = () => {
   const location = useLocation();
-  const [search, setSearch] = useState("");
+  // const [search, setSearch] = useState("");
```
Comment (rishbitsnbytes) [inline src/components/Header.jsx:10]: commented code can be removed if not to be used later!

**Review by xii-vi — APPROVED** [review]
- Product Card Looks great
- Just remove unused code instead of commenting them.

**Review by rishbitsnbytes — APPROVED** [review]
Looks good to merge!

**Comment by TejasShekar** [comment]
> - Product Card Looks great
> - Just remove unused code instead of commenting them.

They are work in progress. So, had to comment them out for this PR.

**Pull request merged**